### PR TITLE
Replace "Overriding" by "Configuring" for ProductValue class

### PR DESCRIPTION
**Pull request by LaureBro**
following my previous PR.. #733

### CODE REVIEW

File: manipulate_pim_data/catalog_structure/creating_a_reference_data.rst [hunk 1]
```diff
@@ -174,13 +174,12 @@ To check if the entities are correctly set up, use the following command:
...
 
-Overriding the ProductValue
+Configuring the ProductValue
 ---------------------------
 
 Depending on the needs, a product can be linked to several colors or just one.
 The first case will be called *simple reference data*, while the second one will be referred to as *multiple reference data*.
 
-To link a reference data to the product, the `ProductValue` object needs to be overriden.
 This task is documented here :doc:`overriding_the_orm_product_value`.
```
Comment (jmleroux) [inline manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: Kill this one too...

Comment (LaureBro) [reply manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: done :)

Comment (jmleroux) [reply manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: Not done, but it's my fault. I was not clear enough.You must remove `This task is documented here :doc:`overriding_the_orm_product_value`.`

File: manipulate_pim_data/catalog_structure/creating_a_reference_data.rst [hunk 2]
```diff
@@ -174,15 +174,12 @@ To check if the entities are correctly set up, use the following command:
...
+Configuring the ProductValue
 ---------------------------
 
 Depending on the needs, a product can be linked to several colors or just one.
 The first case will be called *simple reference data*, while the second one will be referred to as *multiple reference data*.
 
-To link a reference data to the product, the `ProductValue` object needs to be overriden.
-This task is documented here :doc:`overriding_the_orm_product_value`.
-
 Don't forget to check the mapping of the product value and to register the custom class in the container.
```
Comment (SamirBoulil) [inline manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: shouldn't you remove this also ?

Comment (SamirBoulil) [reply manipulate_pim_data/catalog_structure/creating_a_reference_data.rst]: ah maybe not considering JM's message.

**Pull request merged**